tests/test_action_space.py

```
@@ -0,0 +1,59 @@
+
+def test_continuous_space_defaults_to_float() -> None:
+    space = ContinuousSpace(low=0.0, high=1.0)
+    assert space.low == 0.0
+    assert space.high == 1.0
+    assert space.dtype == "float"
+
+
+def test_continuous_space_coerces_int_bounds_and_keeps_int_dtype() -> None:
+    space = ContinuousSpace(low=0, high=200, dtype="int")
+    assert space.dtype == "int"
+    assert isinstance(space.low, float) and isinstance(space.high, float)
```

Ruff (0.15.18)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)